docs/pyodide/index.html

```diff
@@ -1,142 +1,73 @@
-
-    <style>
-        html {
-          overflow-y: scroll;
-          overflow-x: scroll;
-        }
-
-        .demoContainer {
-          display: flex;
-          margin-left: 1em;
-        }
-
-        #text-editor {
-            width: 700px;
-            height: 600px;
-            float: left;
-            margin-bottom:1em;
-            margin-right:1em;
-        }
-
-        .display-none {display:none;}
-    </style>
-  </head>
-
-  <body>
-<p style="font-family: 'sans'; font-size:18px"><b>pyp5js</b> demo editor</p>
-
-    <div class="demoContainer">
-      <div style="display: block">
-          <textarea class="display-none" id="id_py_code" name="py_code"></textarea>
-
-          <div id="text-editor"></div>
-          <div class="clearfix">
-            <input  id="executeBtn" class="btn rounded py1 px2 mb1" type="submit" value="Run" title="(Ctrl/Cmd + Enter) to run the sketch">
-            <input id="clearBtn" class="btn rounded py1 px2 mb1" type="submit" value="Clear" title="(Ctrl/Cmd + .) to clear the sketch">
-          </div>
-      </div>
-
-      <div id="sketch-holder">
-        <!-- You sketch will go here! -->
+<html lang="en">
+
+<head>
+
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>pyp5js with Pyodide Demo</title>
+
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/styles/default.min.css">
+  <script src="https://cdn.jsdelivr.net/npm/p5@1.0.0/lib/p5.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/highlight.min.js"></script>
+  <script src="https://cdn.jsdelivr.net/pyodide/v0.18.1/full/pyodide.js"></script>
+  <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js"
+    integrity="sha256-4+XzXVhsDmqanXGHaHvgh1gMQKX40OUvDEBTu8JcmNs=" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/10.3.2/styles/default.min.css">
+  <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/10.3.2/highlight.min.js"></script>
+
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/ace/1.4.12/ace.js"
+    integrity="sha512-GZ1RIgZaSc8rnco/8CXfRdCpDxRCphenIiZ2ztLy3XQfCbQUSCuk8IudvNHxkRA3oUg6q0qejgN/qqyG1duv5Q=="
+    crossorigin="anonymous"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/ace/1.4.12/mode-python.min.js"
+    integrity="sha512-2Ke4vMGrMfYRM55pT1aA5bw7Pl82Sc7K5Hg8XZYZu+EQrb0AO1mNYTagwZm+MFVAImYS9Mlnm73zcgc01wPXxA=="
+    crossorigin="anonymous"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/ace/1.4.12/theme-monokai.min.js"
+    integrity="sha512-S4i/WUGRs22+8rjUVu4kBjfNuBNp8GVsgcK2lbaFdws4q6TF3Nd00LxqnHhuxS9iVDfNcUh0h6OxFUMP5DBD+g=="
+    crossorigin="anonymous"></script>
+  <script src="share.js"></script>
+  <script src="target/target_sketch.js" type="module"></script>
+
+  <script>hljs.initHighlightingOnLoad();</script>
+
+  <link rel="stylesheet" href="styles.css">
+</head>
```

Comment:
It would be nice to have the `docs/pyodide/index.html.template` updated as well because this `index.html` is generated using that template with the `make refresh_demo` command.

Reply:
Think I did it in d1cf2e2 but I might've missed something.